app/containers/LoginPage/components/TermsNotAccepted.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useMemo } from 'react';
+import { Formik } from 'formik';
+import { useIntl } from 'react-intl';
+import { useDispatch } from 'react-redux';
+
+import { RoutePath } from 'global/constants';
+import { termsAcceptRequest, termsAcceptSaga } from 'global/reducers/auth';
+
+import FormikInput from 'components/FormikInput';
+import LinkButton from 'components/Button/LinkButton';
+import Button from 'components/Button';
+import Row from 'components/Row';
+import TermsCheckbox from 'components/TermsCheckbox';
+import ErrorAlert from 'components/ErrorAlert';
+import FormikForm from 'components/FormikForm';
+
+import { useInjectSaga } from 'redux-injectors';
+import { generateTermsValidationSchema } from '../utils';
+import messages from '../messages';
+
+type Props = {
+  goBack: () => void;
+  error: string;
+  loading: boolean;
+  termsExtraFields: {
+    email: string;
+    firstName: Nullable<string>;
+    lastName: Nullable<string>;
+  };
+};
+
+const TermsNotAccepted = ({
+  error,
+  goBack,
+  termsExtraFields,
+  loading,
+}: Props) => {
+  const { formatMessage } = useIntl();
+  const dispatch = useDispatch();
+
+  useInjectSaga({ key: 'termsAcceptSaga', saga: termsAcceptSaga });
```

Comment:
Please create WithSaga object like in `app/global/reducers/intervention/sagas/fetchCurrentUserCollaboratorData.ts` and use it here
```suggestion
  useInjectSaga(withTermsAcceptSaga);
```
 because last time I spend 2 hours looking for a duplicated saga injection with different keys causing doubling the requests  😆